client/landing/stepper/declarative-flow/internals/steps-repository/goals/select-goals.tsx

```diff
@@ -45,9 +45,11 @@ export const SelectGoals = ( {
 }: SelectGoalsProps ) => {
 	const translate = useTranslate();
 	const goalOptions = useGoals( displayAllGoals );
-	const [ isBuiltByExpressExperimentLoading ] = useExperiment(
+	let [ isBuiltByExpressExperimentLoading ] = useExperiment(
```

Comment:
I like the idea of keeping the experiment skeleton! Let's remove the `useExperiment` calls, as they would cause unnecessary network requests to the Explat endpoint.